content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -0,0 +1,180 @@
+
+You need [read and write variables permissions](/docs/cloud/users-teams-organizations/permissions.html#general-workspace-permissions) to create and edit both workspace-specific variables and variable sets through:
+
+- The Terraform Cloud UI, as detailed below.
+- The Variables API for [workspace-specific variables](/docs/cloud/api/workspace-variables.html) and [variable sets](/docs/cloud/api/variable-sets.html).
+- The `tfe` provider [`tfe_variable`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/variable) resource, which can be more convenient for large numbers of complex variables.
+
+## Workspace-Specific Variables
+
+To view and manage a workspace's variables, go to the workspace and click the **Variables** tab.
+
+The **Variables** page appears, showing all workspace-specific variables and variable sets applied to the workspace. This is where you can add, edit, and delete workspace-specific variables. You can also add and remove variable sets from this workspace.
+
+![Screenshot: The initial appearance of a workspace's variables page](./images/vars.png)
+
+
+### Add a Variable
+
+1. Go to the workspace **Variables** page and click **+ Add variable**.
+
+2. Choose a variable category (Terraform or environment), optionally mark the variable as [sensitive](#sensitive-values), and enter a variable name, value, and optional description.
+
+    Refer to [variable values and format](#variable-values-and-format) for variable limits, allowable values, and formatting.
+
+3. Click **Save variable**. The variable appears in the list of variables for the workspace and Terraform Cloud will begin applying it to runs.
+
+  ![Screenshot: A variable being edited](./images/vars-edit.png)
+
+### Edit a Variable
+
+1. Click the ellipses next to the variable you want to edit and select **Edit**.
+2. Make any desired changes and click **Save variable**.
+
+
+### Delete a Variable
+
+1. Click the ellipses next to the variable you want to delete and select **Delete**.
+2. Type the variable name into the dialog box to confirm and click **Delete**.
+
+## Variable Sets
+
+> **Hands On:** Try the [Create and Use a Variable Set](https://learn.hashicorp.com//tutorials/terraform/cloud-create-variable-set) tutorial on HashiCorp Learn.
+
+To view variable sets for your organization, click **Settings** in the top menu bar and then click **Variable sets** in the left sidebar.
+
+The **Variable sets** page appears, showing a list of all of the variable sets within the organization. Click each variable set to open it and review details about its variables.
+
+![The variable sets page in the Terraform Cloud UI](link)
+
+### Create Variable Sets
+
+-> **Note:** Variable sets are currently in beta.
+
+To create a variable set:
```

Comment:
I like how you prefaced the numbered steps with this. Can we do the same with the workspace-specific variables?

Reply:
yes! thank you - good call - pushed up a change for that!